ui/app/tests/Tests.elm

```diff
@@ -16,3 +18,10 @@ utils =
     describe "Utils"
         [ Filter.all
         ]
+
+
+match : Test
```

Comment:
For me this would rather be called `groupBar`. If you compare it to the example above: `Utils` is the folder and `Filter` is the actual file. Not very important.